assets/js/base/components/cart-checkout/totals/totals-coupon-code-input/index.js

```diff
@@ -21,7 +21,7 @@ import './style.scss';
 const TotalsCouponCodeInput = ( {
 	instanceId,
 	isLoading = false,
-	initialOpen,
+	initialOpen = false,
```

Comment:
why are we closing this by default?

Reply:
It should be closed by default like on frontend. In the editor it was visible.